ongs/utils.py

```diff
@@ -0,0 +1,12 @@
+def check_cnpj_mask(c: str):
```

Comment:
couldn't we just use a regex here? i think this one will do `(^\d{2}\.\d{3}\.\d{3}\/\d{4}\-\d{2}$)`

Reply:
Yeah thanks! I don't know much about regex, but this one did the trick...